docs/helpers/index.rst

```diff
@@ -40,7 +40,7 @@ We now open a sequential file object for writing::
     >>> from baseband.helpers import sequentialfile as sf
     >>> filenames = ["seqvdif_{0}".format(i) for i in range(2)]
     >>> file_size = fh.fh_raw.seek(0, 2) // 2
-    >>> fwr = sf.open(filenames, mode='wb', file_size=file_size)
+    >>> fwr = sf.open(filenames, mode='w+b', file_size=file_size)
```

Comment:
Here and elsewhere, is the change to 'w+b' needed? It means read/write, while for most formats the reading is not needed.

Reply:
I just found it slightly easier to have the checks

```
        if isinstance(name, sf.SequentialFileBase):
            assert (('r' in mode and name.mode == 'rb') or
                    ('w' in mode and name.mode == 'w+b')), (
                        "open only accepts sequential files opened in 'rb' "
                        "mode for reading or 'w+b' mode for writing.")
```
in the generic opener rather than DADA and GUPPI formats.  I suppose I can do:

```
        if isinstance(name, sf.SequentialFileBase):
            assert (('r' in mode and name.mode == 'rb') or
                    ('w' in mode and name.mode in ('wb', 'w+b'))), (
                        "open only accepts sequential files opened in 'rb' "
                        "mode for reading or 'wb' mode for writing.")
```
and do a similar, specific check for for 'w+b' in DADA and GUPPI wrappers, but I don't see it as a hindrance to use a general 'w+b' check.

Reply:
It is mostly that it is surprising that `w+b` is required at all. But let's just leave it as you have it until someone complains!